Kiss.Bff.EndToEndTest/NieuwsEnWerkInstructies/NieuwsEnWerkInstructiesScenarios.cs

```diff
@@ -393,19 +393,35 @@ public async Task Scenario14()
     {
         await Step("Given a unique text (uuid)");
 
-        await Step("Given there is exactly one nieuwsbericht with that text as the title");
+        var uniqueTitle = Guid.NewGuid().ToString();
 
+        await Step("Given there is exactly one nieuwsbericht with that text as the title");
+        
+        await Page.CreateBericht(new() { Title = uniqueTitle, BerichtType = BerichtType.Nieuws });
```

Comment:
@felixcicatt it seems like the berichten are not removed (disposed), what's the right way to make sure they are gone after the test is done?

Reply:
See my comment [here](https://github.com/Klantinteractie-Servicesysteem/KISS-frontend/pull/991#discussion_r1908774829). I should have documented this better.
```suggestion
        await using var nieuws1 = await Page.CreateBericht(new() { Title = uniqueTitle, BerichtType = BerichtType.Nieuws });
```

Reply:
i have resolved comment and pushed a fix